sdk/python/tests/integration/registration/test_cli_apply_duplicated_featureview_names.py

```diff
@@ -45,6 +45,49 @@ def test_cli_apply_duplicated_featureview_names() -> None:
         )
 
 
+def test_cli_apply_imported_featureview() -> None:
```

Comment:
Should we add a test that ensures that duplicate names with different contents raises an exception?

Reply:
There's already a test for such a case

Reply:
done